lib/matplotlib/markers.py

```diff
@@ -393,10 +393,7 @@ def _set_mathtext_path(self):
 
         # again, the properties could be initialised just once outside
         # this function
-        # Font size is irrelevant here, it will be rescaled based on
-        # the drawn size later
-        props = FontProperties(size=1.0)
```

Comment:
The original intention has apparently been to create a text path with size 1 and rescale it later. This was not working because of the incorrect parameter name.

Are we sure that a default `FontProperties` object, which is using `rcParams['font.size']` is the correct way rather than the original fontsize 1?

Reply:
The proposed fix in this PR matches the previous behavior.
I guess the original intent was to use a font size of 1, but I'm not sure this actually makes sense because good quality fonts should look different at very small sizes (e.g. https://en.wikipedia.org/wiki/Font#Optical_size) and we probably won't end up using the textpath at a size close to 1 (not that I think any mainstream font ships with a specific 1-size design either...).
Having the result depend on the rc value may not be optimal either (which has already been the behavior so far), but so far no one complained...

Reply:
Just tested, varying `rcParams['font.size']` doesn't seem to have an effect. At least not something i can see with the bare eye when switching between two images. Maybe the size is not used after all.

Anyway, this is ok then.